reorg

StarfallProjects opened the pull request:
This PR:
- Adds to the README and CONTRIBUTING docs
- Reorganizes the documentation to take advantage of tabs
- Updates all links within the docs
- Includes some proofreading work, particularly starting to standardize headings and UI element references [wip]
- Adds a brief introduction to CodeSee Local

pchocho commented:
Hi, I like the reorganization into tabs. We did intentionally remove the TOC and set a max-width (feedback from Julia that the right column made the page UX confusing), but I think I'm fine with how this is looking now. It's just odd when there isn't much content on a page, or when the headers don't feel like TOC headers.

I had added an &nbsp; at the end of the content of each page to add extra margin to the bottom. It feels too close to the bottom nav now. If we could do this via CSS instead that would be great.

StarfallProjects commented:
> I had added a non-breaking space at the end of the content of each page to add extra margin to the bottom. It feels too close to the bottom nav now. If we could do this via CSS instead that would be great.

Yup 100% - I removed it because expecting folks to remember every time they create a page is always risky. Should be easy to do with CSS. Will do that as part of this PR.

ramonh commented on mkdocs.yml:
```diff
@@ -1,34 +1,38 @@
-    - Configuration options: options.md
+  - 👋 Welcome:
+    - Introduction: index.md
+    - 🙏 Troubleshooting: troubleshooting.md
```
Shall we remove this emoji to keep the subpages' not having them consistent?

StarfallProjects replied:
Sounds good. Removed.

ramonh reviewed (approved):
Looks great @StarfallProjects thank you so much!

Left more of an open-ended question, but lgtm!

ramonh commented:
Nice one, thank you so much!
